explorer/interpreter/interpreter.cpp

```diff
@@ -502,6 +502,11 @@ auto Interpreter::EvalAssociatedConstant(
 auto Interpreter::InstantiateType(Nonnull<const Value*> type,
                                   SourceLocation source_loc)
     -> ErrorOr<Nonnull<const Value*>> {
+  if (trace_stream_->is_enabled()) {
```

Comment:
Could we have a test in `testdata/trace` to cover this?

Reply:
I tried modifying the `testdata/trace/phase_execution.carbon` to include type instantiation.
Let me know you have a better idea.